### ISSUE TITLE
Exclusions and bonds don't work on ParticleSlice

### ISSUE THREAD

**Issue by fweik** [espressomd/espresso#1563]
Because they can have variable length, so they can not be cast into a numpy array. If this can't be fixed, they have to be removed from the slice.

```python
from __future__ import print_function
import espressomd as es

s = es.System()
s.part.add(pos=[0,0,0])
s.part.add(pos=[0,0,0], exclusions=[0])
s.part.add(pos=[0,0,0])

print(s.part[:].exclusions)
```

**Comment by RudolfWeeber** [comment]
IIRC, Tobias Rau implemented these. We need to check whether they got removed by accident when the auto-generated properties were introduced.

**Comment by KonradBreitsprecher** [comment]
Can be closed via #1652

```
from __future__ import print_function
import espressomd as es
s = es.System()
s.box_l = [10,10,10]

fene = es.interactions.FeneBond(k=1, d_r_max=1, r_0=1)
s.bonded_inter.add(fene)

s.part.add(pos=[0,0,0])
s.part.add(pos=[0,0,1], exclusions=[0])
s.part.add(pos=[0,0,2], vs_relative = (0, 5.0, (0.5, -0.5, -0.5, -0.5)) )
s.part[0].add_bond((fene, 1))

print(s.part[:].exclusions)
print(s.part[:].bonds)
print(s.part[:].vs_relative)
```
